src/hooks/useLocation.js

```diff
@@ -24,24 +23,29 @@ const useLocation = () => {
   const [location, setLocation] = useState(INITIAL_COORDS);
 
   useEffect(() => {
+    let unsubscribe = null;
     async function requestLocation() {
       try {
         let granted = await RNLocation.checkPermission(PERMISSIONS);
         if (!granted) {
           granted = await RNLocation.requestPermission(PERMISSIONS);
         }
         if (granted) {
-          const { latitude, longitude } = await RNLocation.getLatestLocation({ timeout: TIMEOUT });
-          setLocation({ latitude, longitude });
+          unsubscribe = RNLocation.subscribeToLocationUpdates(location => {
+            const { latitude } = location[location.length - 1];
+            const { longitude } = location[location.length - 1];
```

Comment:
couldn't this be in a same sentence? 

`const { latitude, longitude } = location[location.length - 1];`